sample/Kmd.Logic.FileSecurity.Client.ConfigurationSample/Program.cs

```diff
@@ -74,6 +75,19 @@ private static async Task Run(AppConfiguration configuration)
                 }
 
                 Console.WriteLine("Certificate ID: {0} \nCertificate Name: {1}\nSubscription ID : {2}", result.CertificateId, result.Name, result.SubscriptionId);
+
+                var signConfigurationID = configuration.SignConfigurationDetails.SignConfigurationId;
+
+                Log.Information("Fetching sign configuration details for sign configuration id {SignConfigurationId}", configuration.SignConfigurationDetails.SignConfigurationId);
+                var signConfigurationResult = await fileSecurityClient.GetSignConfiguration(signConfigurationID);
+
+                if (signConfigurationResult == null)
+                {
+                    Log.Error("Invalid sign configuration id {Id}", configuration.SignConfigurationDetails.SignConfigurationId);
```

Comment:
Instead of Invalid sign configuration you can mention Sign configuration not found.

Reply:
made changes as suggested